include/iceflow/consumer-tlv.hpp

```diff
@@ -169,22 +189,30 @@ class ConsumerTlv {
     m_scheduler.schedule(duration, [this] { sendAnchor(); });
   }
 
+  /**
+   * Receives the data from upstreams
+   * Pushes the data to the Input Queue of the Consumer
+   * Increase the data fetching/Interest sending rate - AIMD Congestion Control
+   * @param interest named data interest
+   * @param data actual data objects
+   */
   void onData(const ndn::Interest &interest, const ndn::Data &data) {
+    //    ndn::Block cont;
     ndn::Block cont;
     uint32_t contentType = data.getContentType();
     if (data.hasContent()) {
 
       switch (contentType) {
-
-      case MainData: {
+        // Main Data here was the Manifest. Manifest is the collection of names
+        // of the data objects
```

Comment:
Should be documented in the enum value or for a separate handler function but not here IMHO.

```suggestion
```